src/bsim/winter2021/Bacterium.java

```diff
@@ -69,6 +73,8 @@ public Bacterium(BSim sim, Vector3d px1, Vector3d px2, long origin_id, long pare
         this.origin_id = origin_id;
         this.parent_id = parent_id;
         lifetime = 0;
+        toxintime=0;
+        contact=0;        

```

Comment:
Remove these from Bacterium.java; these should only be associated with certain types of bacteria